packages/react-tabs/src/components/Pivot/examples/Pivot.Basic.Example.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
-import { Label, ILabelStyles } from '@fluentui/react-next/lib/Label';
```

Comment:
We should add `react-tabs` to PR deploy site, similar to button: https://github.com/microsoft/fluentui/pull/13873